### PR TITLE
Update VAnchor Withdraw UI

### PR DESCRIPTION
**Pull request by AtelyPham**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

The withdraw UI only accepts a note input at the moment. And the relayer selection is shown on the modal.
 
Issue Number: Closes https://github.com/webb-tools/webb-dapp/issues/397

## What is the new behavior?

- User can add additional notes to withdraw.
- Each note with have its corresponding info (destination chain and amount available).
- Move the relayer selection input to the main withdraw view when the note is provided (previously user had to click the relayer button to select relayer in a pop-up modal).

https://user-images.githubusercontent.com/60747384/175991934-8f86e101-decf-42b9-9855-2e3c253116c0.mp4

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

## Other information
We currently accept multiple' notes, and each note has a destination chain. But in the mockup UI, we don't have any input for selecting the destination chain.

I'm wondering which destination chain we will choose if the inputted notes have different destination chains.

### CODE REVIEW

**Review by nepoche — CHANGES_REQUESTED** [review]
One small thing I would change: can we have the note input extend to fill the container - and shrink it when the note is parsed to display information on the right side? It all looks great though, nice job!

File: packages/vbridge/src/hooks/withdraw/useWithdraw.tsx [hunk 1]
```diff
@@ -55,56 +64,125 @@ export const useWithdraw = (params: UseWithdrawProps) => {
...
+          const noteErrors = p.validationError.notes.slice();
+          noteErrors.push(validationError.note);
+
+          return {
+            ...p,
+            validationError: {
+              notes: noteErrors,
+              recipient: validationError.recipient,
+            },
+          };
```
Comment (AtelyPham) [inline packages/vbridge/src/hooks/withdraw/useWithdraw.tsx:181]: I'm not sure whether it's good or not when I handle note errors like this. Don't know how to test by the way 😢

**Comment by drewstone** [comment]
Merge conflicts here @AtelyPham

**Review by nepoche — APPROVED** [review]

File: packages/vbridge/src/hooks/withdraw/useWithdraw.tsx (hunk 1 above)
Comment (AhmedKorim) [reply packages/vbridge/src/hooks/withdraw/useWithdraw.tsx:181]: @AtelyPham  it's okay we can spec that next

**Pull request merged**